### DISCUSSION TITLE
Certificate available but cannot enforce HTTPS from pages

### DISCUSSION THREAD

**Discussion by notsag-dev** [community/community#43666] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi everybody! I have an issue with my github page: https://www.nfcogos.com which is hosted in the repo https://github.com/notsag-dev/landing-nfcogos-pages

If you access it including https in from of the domain you will see it's accessible securely so the certificate was generated successfully. But when I go to the pages settings page it doesn't allow me to enforce HTTPS, I keep seeing the "Certificate Request Error: Certificate provisioning will retry automatically in a short period, please be patient. " error. Additionally, when I enter www.nfcogos.com it will display the insecure version. 

This may have been caused because before this domain was related to another repo, which I unpublished and from which I removed the CNAME file so it should be clean now.

Any ideas on how to clean this up from it to allow me to enforce HTTPS?

Thank you.

**Comment by notsag-dev** [comment, marked as answer]
I was stuck for like 2 days and then started working again. I guess it took a bit more because I changed the repo this domain was associated to. It's working properly now

**Comment by duyen2310** [comment]
Did you figure it out?
